src/components/Tab/Tab.tsx

```diff
@@ -39,7 +39,6 @@ export const Tab = ({
   className,
   id,
   'aria-labelledby': ariaLabelledBy,
-  title,
```

Comment:
Removing this allows `title` to fall thru to the `...other`, which means we now can support title attributes on tabs (before, this named prop was not being used below, preventing the title from actually showing).

updated the snapshot to cover this from regressing.

Reply:
The only effect this has is the little popover text when a mouse user hovers over the tabbed panel element, right?

Reply:
@dierat correct. this change is mostly to support what some of our snapshots were testing (we had a snapshot specifying title but not rendering it, which was a bug). I don't think we wanna rely on `title` in general, tho... it's apparently a bit iffy on the a11y side of things

Reply:
In this case, the `aria-labelledby` attribute is overriding `title` for screen readers anyway, so I'm not concerned about it reducing the experience. What I'm wondering is whether we need it at all or if it should be nixed with the others since it's not adding much.

Reply:
yah, from what I can glean, the snapshot only incidentally is specifying `title` so we  could also specify some other attribute supported on the tag just to demonstrate `...other` is doing what it should. 

There is no longer code directly specifying `title` so this already nixes that part

Reply:
>we could also specify some other attribute supported on the tag just to demonstrate ...other is doing what it should

I don't think we need a test for the `...other` functionality. You can add something (like a `data-testid` attribute) if you want, but I'm also fine with just removing the `title` and moving on.